server/webapp/WEB-INF/rails.new/app/assets/new_javascripts/views/pipeline_configs/jobs_config_widget.js.msx

```diff
@@ -228,6 +240,12 @@ define([
                 selectedIndex={ctrl.vmState('selectedJobIndex', m.prop(0))}>
           {ctrl.map(function (job) {
             var jobIndex = args.jobs().indexOfJob(job);
+            var isRunOnAllAgents = job.isRunOnAllAgents();
+            var elasticProfileIdMessage;
+            if (isRunOnAllAgents) {
+              elasticProfileIdMessage = 'Elastic profile id not allowed to set for a job with "Run on all agents". Please check "Run on one agent" or "Run instances of job".';
```

Comment:
Radio boxes can be "selected" not "checked". So the message need to be `Please select one of "Run on..." or "Run instances..."`